server/index.js

```diff
@@ -29,11 +29,19 @@ app.set('view engine', 'ejs')
 app.use(setupBodyParsers())
 app.use(setupHealthChecks())
 app.use(setupWebSecurity())
+
+app.use('/client.js', express.static(path.join(__dirname, '../build/client.js')))
```

Comment:
because there is no `static` folder for new nomis we effectively have to map each item at the root, but ignore `index.html` which we still need to be authenticated